src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static isAlive() {
```

Comment:
`isAlive` method should return true or false, but in your case, you filter the array. Let's rename this method